frontend/tests/badges/BadgesTest.php

```diff
@@ -165,17 +165,13 @@ public function testAllBadges() {
     }
 
     public function testListBadges() {
-        // Manually get all badges
-        $aliases = array_diff(scandir(static::OMEGAUP_BADGES_ROOT), ['..', '.', 'default_icon.svg']);
-        $badges = [];
-        foreach ($aliases as $alias) {
-            if (is_dir(static::OMEGAUP_BADGES_ROOT . "/${alias}")) {
-                $badges[] = $alias;
-            }
-        }
+        // Manually creates a new badge
+        $newBadge = 'testBadge';
+        $newBadgePath = static::OMEGAUP_BADGES_ROOT . "/${newBadge}";
+        mkdir($newBadgePath);
```

Comment:
ándale, este approach me gusta más! sólo necesita un poco más de robustez y listo.

```suggestion
        try {
            mkdir($newBadgePath);
            $results = BadgesController::apiList(new Request([]));
        } finally {
            @rmdir($newBadgePath);
        }
```

Reply:
Ya hice las correcciones!